doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -143,14 +143,16 @@ management.
 > This chapter should describe considerations about hardware acceleration, like
 device management.
 
-<a name="3.2.1.4"></a>
-#### 3.2.1.4 CPU management
+<a name="3.2.1.2"></a>
+#### 3.2.1.2 CPU Management
 
-> This chapter should describe considerations about CPU management.
+CPU management has policies to determine placement preferences to use for workloads that are sensitive to cache affinity or latency, so must not be moved by OS scheduler or throttled by kubelet. Additionally, some workloads are sensitive to differences between physical cores and SMT, while others (like DPDK workloads) are designed to run on isolated CPUs (like Linux isolcpus kernel parameter specifying cores isolated from general SMB balancing and scheduler algorighms).
```

Comment:
It is confusing as to what should "not be moved"?
For DPDK, in addition to isolcpus should cset be mentioned also?
Minor typo: SMB instead of SMP
```suggestion
CPU management has policies to determine placement preferences to use for workloads that are sensitive to cache affinity or latency, and so the workloads must not be moved by OS scheduler or throttled by kubelet. Additionally, some workloads are sensitive to differences between physical cores and SMT, while others (like DPDK workloads) are designed to run on isolated CPUs (like Linux isolcpus kernel parameter specifying cores isolated from general SMP balancing and scheduler algorithms).
```

Reply:
> 
> 
> It is confusing as to what should "not be moved"?
> Minor typo: SMB instead of SMP

Thanks for clarifying and fixing.

> For DPDK, in addition to isolcpus should cset be mentioned also?

Added "with cpuset-based selection of CPUs".